src/components/posts/ConclusionGraphic.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import "./conclusion.css";
+import conclusion from "../../assets/conclusion.js";
+export default function ConclusionGraphic() {
+
+  const [currentCard, setCurrentCard] = useState(0);
+
+  const currentCardHandler = (dir) => {
```

Comment:
Super minor point, but I think two functions, `nextCard` and `previousCard`, or some `shiftCard` function may be cleaner. I think adding the `"back"` and `"forward"` strings adds just another set of complexity.

You also may want to consider adding some sort of safety check - I know this sounds silly since you wrote all of the code, but some check so that you don't go out of the bounds of the array makes it more maintainable!